### PR TITLE
Add EIP: Fork-choice enforced Inclusion Lists (FOCIL)

### PR DESCRIPTION
**Pull request by soispoke**
This EIP proposes a committee-based, Fork-choice enforced Inclusion Lists (FOCIL) mechanism to preserve Ethereum’s censorship resistance properties by guaranteeing timely transaction inclusion.

### CODE REVIEW

File: EIPS/eip-FOCIL.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-FOCIL.md:1]:
```suggestion
---
eip: 7805
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: EIPS/eip-FOCIL.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+title: Committee-based, Fork-choice enforced Inclusion Lists (FOCIL)
+description: Allow a committee of validators to force-include a set of transactions in every block
+author: Thomas Thiery <thomas.thiery@ethereum.org>, Francesco D'Amato <francesco.damato@ethereum.org>, Julian Ma <julian.ma@ethereum.org>, Barnabé Monnot <barnabe.monnot@ethereum.org>, Terence Tsao <ttsao@offchainlabs.com>, Jacob Kaufmann <jacob.kaufmann@ethereum.org>, Jihoon Song <jihoonsong.dev@gmail.com>
+
+discussions-to: <URL>
```
Comment (abcoathup) [inline EIPS/eip-FOCIL.md:6]: Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5

File: EIPS/eip-7805.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-7805.md:1]:
```suggestion
---
eip: 7805
```
Need to add assigned EIP number

**Comment by soispoke** [comment]
@abcoathup Thanks for reviewing, the only remaining errors seem to be due to links I included, not sure how I should proceed for those (I do want to keep them in the EIP).

**Pull request merged**